test/sqllogictest/select_introspection.slt

```diff
@@ -81,18 +80,30 @@ DROP CLUSTER REPLICA test.replica_b;
 statement ok
 SELECT * FROM mz_internal.mz_compute_exports
 
-# Verify that querying introspection data is disallowed on clusters with
-# introspection disabled.
+# Verify that querying introspection data is disallowed on replicas with
+# introspection disabled, but allowed on introspection-enabled replicas
+# in the same cluster.
 
 statement ok
 DROP CLUSTER test CASCADE
 
 statement ok
 CREATE CLUSTER test
-  REPLICAS (replica_a (SIZE '1')),
-  INTROSPECTION INTERVAL 0
+  REPLICAS (
+    replica_a (SIZE '1', INTROSPECTION INTERVAL 0),
+    replica_b (SIZE '1', INTROSPECTION INTERVAL 1)
```

Comment:
Is a naked number without a specifier interpreted as seconds or something else? Maybe all values except 0 should be disallowed.

Reply:
Numbers are interpreted by casting them to a string and `INTERVAL '1'` means one second.

```
materialize=> SELECT INTERVAL '1';
 interval
----------
 00:00:01
(1 row)
```

I agree that this is not obvious, but it is consistent with Postgres. We could think about not doing the implicit cast from number to string though.